02week/pigLatin.js

```diff
@@ -3,49 +3,58 @@
 const assert = require('assert');
 const readline = require('readline');
 const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
-
-function pigLatin(word) {
-
-  // Your code here
-
+function pigLatin(str) {
+
+    let fullWord = str.toLowerCase().trim(); // clean up the input
```

Comment:
nice concise code, but does not handle the case where there is no vowels such as fry.